cpu/sam0_common/periph/gpio.c

```diff
@@ -92,6 +92,14 @@ void gpio_init_mux(gpio_t pin, gpio_mux_t mux)
     port->PMUX[pin_pos >> 1].reg |=  (mux << (4 * (pin_pos & 0x1)));
 }
 
+void gpio_disable_mux(gpio_t pin)
```

Comment:
Should this be wrapped with `#ifdef MODULE_PERIPH_I2C_RECONFIGURE`

Reply:
No, this can be useful on it's own (and it's internal to sam0)